starter-code/iteration2/iteration2/src/app/app.component.html

```diff
@@ -0,0 +1,6 @@
+<!doctype html>
+<div>
+  <h1>{{title}}</h1>
+  <img src="{{image}}" alt="" width="300">
+  <p>{{description}}</p>
+</div>
```

Comment:
you should add the selector here instead of the index.html (if it's still not working, come to us and we give it a look)